crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -242,10 +245,16 @@ impl Starknet {
         // set block timestamp and context block timestamp for contract execution
         let block_timestamp = match timestamp {
```

Comment:
We probably don't need the `timestamp` parameter passed in; can it be refactored?

Reply:
Fixed in 35a472416c51ca9d797ad6c7fc3c5eb611ecf9a3